build-in-docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+
+set -e
+
+mkdir -p /catalogue-data/htmlcache
+mkdir -p /catalogue-data/lucene
```

Comment:
This doesn't match with the directory createn during the image build (`/var/local/gn_data`)